pokemon/documentation.md

```diff
@@ -0,0 +1,32 @@
+
+import pandas as pd
+
+## Step 1- Data Munging
+
+# Storing ‘combats.csv(win/lose data)’ file data into a dataframe variable data
+data = pd.read_csv('./pokemon-challenge/combats.csv')
+
+# To replace winner/loser pokemon id with 0 and 1, loop over data rows-
```

Comment:
Minor consistency point - perhaps unnecessary "-" at end of this line.  Recommend change this or else bring every other sub-heading into line with it.